src/features/page_view_timing/aggregate/index.js

```diff
@@ -113,7 +114,7 @@ export class Aggregate extends AggregateBase {
       attrs.cls = cumulativeLayoutShift.current.value
     }
 
-    this.timings.add({
+    this.timings.addEvent({
```

Comment:
This PR sort of feels like it is doing a lot of similar things to the central event buffer PR with different syntax, but it does not reinforce the concept that a feature agg class should have a standard way to store event data as opposed to the [central event buffer PR](https://github.com/newrelic/newrelic-browser-agent/pull/1171/files#diff-d4d922997ccbb1ffc77794d3ec5b070975a56d6e5cc7cc1c43887ed0be2127a2R19), since it is not built in as native part of the agg-base class.